apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/ExplanationOfBenefitResourceProviderIT.java

```diff
@@ -1863,7 +1770,7 @@ public void searchEobWithLastUpdated() throws FHIRException {
 
     // Build up a list of lastUpdatedURLs that return > all values values
     String nowDateTime = new DateTimeDt(Date.from(Instant.now().plusSeconds(1))).getValueAsString();
-    String earlyDateTime = "2019-10-01T00:00:00+00:00";
+    String earlyDateTime = "2019-10-01T00:00:00-04:00";
```

Comment:
More date stuff that should be kept as UTC per an earlier story; sniff all these out so we can retain our previous fixes.

Reply:
Good catch; one of the (BIG) problems I had was the fact that these code changes along with parallel tracks of changes caused me to constantly merge other's changes...I worried about this sort of case where I missed something from master. Will do a global search on this one and fix.